build/system.gyp

```diff
@@ -11,6 +11,19 @@
   },
   'targets' : [
     {
+      'target_name': 'linux_notification',
+      'type': 'none',
+      'conditions': [
+        ['tizen!=1', {
+          'link_settings': {
+            'libraries': [
+              '<!@(pkg-config --libs libnotify)',
```

Comment:
Please look at all the other targets in this file. You need to check for `building_crosswalk_bin`.